packages/langium/langium-config.json

```diff
@@ -5,7 +5,10 @@
         "grammar": "src/grammar/langium-grammar.langium",
         "fileExtensions": [
           "langium"
-        ]
+        ],
+        "chevrotainParserConfig": {
+            "maxLookahead": 3
```

Comment:
So you want to use LL(k) for the grammar language?

Reply:
It fixes a completion issue, due to very slight differences in lookahead behavior for malformed input.

Reply:
General question, could we allow a different parser config for completion?

Reply:
Not really, since the parser is build the same way, the implementation of the embedded actions are just different.

Reply:
Ok last question, have you checked whether _k=2_ would work for the grammar language?

Reply:
Haven't checked, but it might probably work.

Reply:
Or maybe even _k=1_? 😁